src/routes/SearchPage/index.js

```diff
@@ -48,14 +53,26 @@ class ContentListAdapter extends React.Component {
     const url = normalizeUrl(this.getParentPath(), {removeTrailingSlash: true})
     const pages = []
     this.props.pages.children.forEach(page => this.findPages(pages, url, page))
+
     return <ContentList pages={pages}/>
   }
 }
 
+function mapStateToProps (state) {
+  return {
+    language: state.router.params.language,
+    path: state.router.params['_'] // _ contains all the values from *
+  }
```

Comment:
Why are they not required? 🤔

Reply:
path and language doesn't have to be in the url, they are optional